_includes/utm_cookies.html

```diff
@@ -0,0 +1,17 @@
+<script>
+$(document).ready(function() {
+  if (document.cookie.includes("u_source")) {
+    return;
+  };
+
+  var date = new Date();
+  date.setMonth(date.getMonth() + 6);
```

Comment:
I assume this should work even when month is 11+6 = 17

https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Date/setMonth

Reply:
Yes, and it would update the year in such case.